etc/master.passwd

```diff
@@ -1,2 +1,2 @@
 root::0:0::0:0:root:/root:/bin/ksh
-
+paco::1000:1000::0:0:paco:/root:/bin/ksh
```

Comment:
Is _paco_ a special username? Why not something like _user_ or _mimiker_?
I'm wonder if `/root` is a good home directory for any user other than _root_.

Reply:
For now changed to users 'admin' and 'user'. Home directory for all users is now /